cmd/kube-controller-manager/app/core.go

```diff
@@ -68,8 +68,7 @@ func startServiceController(ctx ControllerContext) (bool, error) {
 		ctx.Options.ClusterName,
 	)
 	if err != nil {
-		glog.Errorf("Failed to start service controller: %v", err)
-		return false, nil
+		return true, fmt.Errorf("error creating Service controller: %v", err)
```

Comment:
This one shouldn't fail.  I don't recall the details, but this was made a glog for some reason.  It lives like this as a legacy.

Reply:
let me revert back this change and I will add a comment